### PR TITLE
update httpclient call to use a response handler

### PR DESCRIPTION
**Pull request by dev-mlb**

### CODE REVIEW

**Review by sambish5 — APPROVED** [review]

File: src/main/java/emissary/client/EmissaryResponse.java [hunk 1]
```diff
@@ -117,5 +119,15 @@ private <T extends BaseEntity> T makeErrorEntity(String msg, Class<T> mapper) {
         return r;
     }
 
+    public static class EmissaryResponseResponseHandler implements HttpClientResponseHandler<EmissaryResponse> {
```
Comment (drivenflywheel) [inline src/main/java/emissary/client/EmissaryResponse.java:122]: Not that it matters, but why not just `EmissaryResponseHandler`?

**Review by drivenflywheel — DISMISSED** [review]

File: src/main/java/emissary/client/EmissaryResponse.java (hunk 1 above)
Comment (dev-mlb) [reply src/main/java/emissary/client/EmissaryResponse.java:122]: Because that would make too much sense :P

Comment (dev-mlb) [reply src/main/java/emissary/client/EmissaryResponse.java]: Done. Updated to ResponseEmissaryResponseResponseHandlerResponse.

**Review by drivenflywheel — APPROVED** [review]
🚀

**Review by sambish5 — APPROVED** [review]

**Pull request merged**